run-setup

```diff
@@ -103,36 +103,27 @@ systemctl reload nginx
 mkdir -p /mnt/heapdumps
 
 install -d -m755 /usr/lib/systemd/system/
-install -C -m644 *.service *.timer *.mount -t /usr/lib/systemd/system/
-systemctl daemon-reload
+install -C -m644 sync-diag.service *.timer *.mount -t /usr/lib/systemd/system/
 
-restart-unit-if-not-masked () {
-	if [[ $# -ne 1 ]]; then
-	    echo "restart-unit-if-not-masked: $#: wrong number of arguments"
-	    return 1
-	fi
-	local unit; unit=$1; shift
+if [[ ${PROFILE:-} = sandboxnet ]]; then
+    apt install --yes --no-install-recommends python3-pip
+    python3 -m pip install --user pipenv
+    /root/.local/bin/pipenv sync
+    install --compare --mode=644 autopropose.py.service --no-target-directory /usr/lib/systemd/system/autopropose.service
```

Comment:
OK, I get that you prefer long option names. But please, at least make it easier for me to read by wrapping the lines at 80th column. I (and others) divide screen estate into several windows.

Reply:
Fair enough.